### PR TITLE
Improve use of `Ref.is_null/valid`

### PR DESCRIPTION
**Pull request by AThousandShips**
Use `is_null` over `!is_valid` and vice versa.

See:
* https://github.com/godotengine/godot/pull/96076

### CODE REVIEW

**Review by Mickeon — APPROVED** [review]
Oh my goodness that's a lot.

File: editor/plugins/script_editor_plugin.cpp [hunk 1]
```diff
@@ -3412,12 +3412,10 @@ void ScriptEditor::_make_script_list_context_menu() {
 	context_menu->add_separator();
 	if (se) {
 		Ref<Script> scr = se->get_edited_resource();
-		if (scr.is_valid()) {
-			if (!scr.is_null() && scr->is_tool()) {
-				context_menu->add_shortcut(ED_GET_SHORTCUT("script_editor/reload_script_soft"), FILE_TOOL_RELOAD_SOFT);
-				context_menu->add_shortcut(ED_GET_SHORTCUT("script_editor/run_file"), FILE_RUN);
-				context_menu->add_separator();
-			}
+		if (scr.is_valid() && scr->is_tool()) {
```
Comment (AThousandShips) [inline editor/plugins/script_editor_plugin.cpp:3415]: Collapsed this as it was just redundant, cause if the mismatch of `+/-`

**Review by Repiteo — APPROVED** [review]
LGTM

**Comment by Repiteo** [comment]
Thanks!

**Comment by AThousandShips** [comment]
Thank *you*!

**Pull request merged**